src/Coordination/Changelog.cpp

```diff
@@ -1895,17 +1911,34 @@ try
             }
             else if (changelog_description.from_log_index > last_read_index && (changelog_description.from_log_index - last_read_index) > 1)
             {
-                if (!last_log_read_result->error)
+                if (changelog_description.from_log_index <= last_commited_log_index)
```

Comment:
I would add a comment with clear explanation: this changelog miss some log entries, but all of them are already commited, so nothing bad is happening here.